src/main/java/blackjack/user/PlayerName.java

```diff
@@ -12,20 +14,34 @@ public PlayerName(final String name) {
     }
 
     private void validateNameSize(final String name) {
-        if (name.isEmpty()) {
+        if (name.isBlank()) {
             throw new IllegalArgumentException("이름은 한 글자 이상 입력해야 합니다.");
         }
     }
 
     private void validateCharacter(final String name) {
-        boolean isNotAllLetter = name.chars().anyMatch(ch -> !Character.isLetter(ch));
+        boolean isNotAllLetter = name.chars().anyMatch(c -> !Character.isLetter(c));
```

Comment:
```suggestion
        boolean isNotAllLetter = name.chars().noneMatch(Character::isLetter);
```
noneMatch를 제안드려 보아요~

Reply:
앗! 그렇네요.. 사나 말씀이 맞아요! 너무 섣불리 생각했네요🥺